scripts/importmcdev.sh

```diff
@@ -41,6 +41,7 @@ import NextTickListEntry
 import ThreadServerApplication
 import BlockChest
 import BlockGrass
+import BlockFlowing
```

Comment:
Shouldn't this be removed?

Reply:
Due to the fashion in which I had to lobotomize the Paper build system, this line is required.

Reply:
So maybe 5th step from there https://github.com/canyonmodded/canyon/blob/master/CONTRIBUTING.md#help-i-cant-find-the-file-im-looking-for should be removed? (or i'm misunderstanding it?)

Reply:
Fifth step is to "edit away" - isn't that the point of doing a patch? That's literally the last item on the list.

Reply:
Ohh, nevermind im dumb